src/entities/Flyer.ts

```diff
@@ -39,9 +39,13 @@ export class Flyer {
   @Property({ type: () => [String] })
   organizationSlugs: [string];
 
+  @Field()
+  @Property()
+  startDate: Date;
+
   @Field()
   @Property({ default: 0 })
-  shoutouts?: number;
+  timesClicked?: number;
```

Comment:
Are we getting rid of shoutouts and just keeping track of the times clicked instead? In the future document something like this in the PR description -- i.e., "Removed shoutouts in lieu of a timesClicked field because we've chosen not to implement shoutouts for flyers."